post-deployment/openstack/opsview-metadata.py

```diff
@@ -52,7 +52,7 @@ def generateJson(domain, api, name, logging, region):
                 }
 
     if logging == True:
-        routeList['kibana-openshift-logging.apps'] = '403' 
+        routeList['kibana-openshift-logging.apps'] = '302' 
```

Comment:
403 is forbidden, 302 is a temporary redirect. Do we know if this is a permanent change which will affect all clusters or it is to do with cluster config? I'm wondering if we should keep both response codes as (oddly) 'valid'? Can we put multiple values here?

Reply:
In 4.6 onwards it changed from 403 to 302 but yeah good idea putting both in. Changed to 302,403 and tested using the troubleshoot function in Opsview - it works.